### ISSUE TITLE
add recommended install instructions on the readme?

### ISSUE THREAD

**Issue by jerabaul29** [OpenDrift/trajan#62]
Adding it on the readme directly may be useful for n00bs like me? :)

```
try latest main again. git clone + pip install -e . to replace the pip package.
```

may be enough, just so that the instructions are easily available? :)

**Comment by jerabaul29** [comment]
Yes, this is where I got the installation instructions last, but:

- I wonder if having this directly on the readme may be useful (though this is just taste / you should decide :) ), so I / users do not have to go through link after link to "just" install :)
- the "install from git" was not listed, only the pypy pip and the conda :) - so could maybe be added there if you want to keep it all there rather than in the readme? :)

**Comment by gauteh** [comment]
Yes, I agree, they should also be in the README. As with OpenDrift we try to guide the different users to the most appropriate method. For most it should be enough to just do either conda install trajan og pip install trajan.
